packages/kit/src/views/Onboardingv2/pages/AddExistingWallet.tsx

```
@@ -1,11 +1,11 @@
-import { useMemo } from 'react';
+import { useCallback, useMemo } from 'react';
 
 import { noop } from 'lodash';
 import { useIntl } from 'react-intl';
 import { StyleSheet } from 'react-native';
 
 import type { IKeyOfIcons } from '@onekeyhq/components';
-import { Dialog, Icon, Page, SizableText, YStack } from '@onekeyhq/components';
+import { Button, Dialog, Icon, Page, SizableText, Stack, YStack } from '@onekeyhq/components';
```

ESLint

[error] Replace `·Button,·Dialog,·Icon,·Page,·SizableText,·Stack,·YStack·` with `⏎··Button,⏎··Dialog,⏎··Icon,⏎··Page,⏎··SizableText,⏎··Stack,⏎··YStack,⏎`

(prettier/prettier)